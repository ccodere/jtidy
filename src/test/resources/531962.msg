<?xml version="1.0" encoding="ISO-8859-1"?>
<!-- expected messages for test 531962-->
<messages>
  <message>
    <code>58</code>
    <level>2</level>
    <line>7</line>
    <column>1</column>
    <text><![CDATA[<img> attribute with missing trailing quote mark]]></text>
  </message>
  <message>
    <code>51</code>
    <level>2</level>
    <line>7</line>
    <column>1</column>
    <text><![CDATA[img attribute "width" has invalid value "10 height="]]></text>
  </message>
  <message>
    <code>49</code>
    <level>2</level>
    <line>7</line>
    <column>1</column>
    <text><![CDATA[img lacks "alt" attribute]]></text>
  </message>
  <message>
    <code>110</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Doctype given is "-//W3C//DTD XHTML 1.0 Strict//EN"]]></text>
  </message>
  <message>
    <code>111</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[InputStream: Document content looks like XHTML 1.1]]></text>
  </message>
  <message>
    <code>-1</code>
    <level>0</level>
    <line>0</line>
    <column>0</column>
    <text><![CDATA[3 warnings, no errors were found!]]></text>
  </message>
  <message>
    <code>1</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[The alt attribute should be used to give a short description
of an image; longer descriptions should be given with the
longdesc attribute which takes a URL linked to the description.
These measures are needed for people using non-graphical browsers.]]></text>
  </message>
  <message>
    <code>112</code>
    <level>0</level>
    <line>1</line>
    <column>1</column>
    <text><![CDATA[For further advice on how to make your pages accessible
see "http://www.w3.org/WAI/GL". You may also want to try
"http://www.cast.org/bobby/" which is a free Web-based
service for checking URLs for accessibility.]]></text>
  </message>
</messages>
