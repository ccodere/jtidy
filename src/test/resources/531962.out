<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>[531962] Closing quotes around attribute values</title>
</head>
<body>
<img src="images/spacer.gif" width="10 height=" />
</body>
</html>
